File: src/day1/MathUtil.java
package day1;

public class MathUtil {

	private MathUtil() {
		super();
	}

	public static int min(int a, int b, int c) {
		if (a < b && a < c) {
			return a;
		} else if (b < c) {
			return b;
		} else {
			return c;
		}
	}

	public static int max(int a, int b, int c) {
		if (a > b && a > c) {
			return a;
		} else if (b > c) {
			return b;
		} else {
			return c;
		}
	}

}

File: demo/04-klassen/mathutil.py

class MathUtil:

    @staticmethod
    def min(a, b, c):
        if a < b and a < c:
            return a
        if b < c:
            return b
        return c    

    @staticmethod
    def max(a, b, c):
        if a > b and a > c:
            return a
        if b > c:
            return b
        return c   


